code/datums/components/squeak.dm

```diff
@@ -1,6 +1,5 @@
 /datum/component/squeak
-	var/static/list/default_squeak_sounds = list('sound/items/toysqueak1.ogg'=1, 'sound/items/toysqueak2.ogg'=1, 'sound/items/toysqueak3.ogg'=1)
-	var/list/override_squeak_sounds
+	var/override_squeak_datum
```

Comment:
the var is named override but there is no default var so such naming may be confusing